packages/docs/docs/components/path.mdx

```diff
@@ -37,7 +37,7 @@ The `Path` node allows us to draw and animate SVG paths.
 `Path` nodes are great for displaying precalculated SVG paths, but the format
 can be confusing. If you want to have more intuitive control of the path, and
 control each point within Motion Canvas, we recommend checking out the
-[`Spline`][spline] component.
+[`Spline`](spline) component.
```

Comment:
The syntax used previously was actually valid, it's called [reference-style](https://www.markdownguide.org/basic-syntax/#reference-style-links). However, the said reference was missing:
```md
[`Spline`][spline]

// This was missing from the bottom of the page
[spline]: /api/2d/components/Spline
```
Your new link still won't work, you'd need to specify the full link:
```md
[`Spline`](/api/2d/components/Spline)
```
or revert to the previous syntax and add the missing reference at the bottom.

Reply:
Apparently my new link still accidentally works since the two files (Path and Spline) just so happen to be in the same folder. I'll fix this in a second by just using the reference style properly